Utilities/DataHelper/DataHelper/constants.h

```diff
@@ -0,0 +1,10 @@
+#ifndef constants_h
+#define constants_h 
+/**
+ * header file to provide constants via namespace
+ */
+namespace constants
+{
+ extern const double twopi;
```

Comment:
Hi @ihepzhangyao and @fucd ,
I think twopi is already defined in CLHEP: CLHEP/Units/SystemOfUnits.h
```c++
  static constexpr double     pi  = 3.14159265358979323846;
  static constexpr double  twopi  = 2*pi;
  static constexpr double halfpi  = pi/2;
  static constexpr double     pi2 = pi*pi;
```

Do you think we could avoid such duplication?
Tao

Reply:
Hi @mirguest 

I fixed some runtime error, and remove constans in DataHelper. It can work with CEPCSW simulation now.

Chengdong

Reply:
Hi @fucd and @ihepzhangyao,
Thanks. Should I merge this PR now?

Tao

Reply:
@mirguest 

Yes, it can be merged now. Thanks.

Chengdong